databuilder/models/user.py

```diff
@@ -0,0 +1,145 @@
+from typing import Union, Dict, Any  # noqa: F401
+
+from databuilder.models.neo4j_csv_serde import (
+    Neo4jCsvSerializable, NODE_KEY, NODE_LABEL
+)
+
+
+class User(Neo4jCsvSerializable):
+    # type: (...) -> None
+    """
+    User model. This model doesn't define any relationship.
+    """
+    USER_NODE_LABEL = 'User'
+    USER_NODE_KEY_FORMAT = '{email}'
+    USER_NODE_EMAIL = 'email'
+    USER_NODE_FIRST_NAME = 'first_name'
+    USER_NODE_LAST_NAME = 'last_name'
+    USER_NODE_FULL_NAME = 'full_name'
+    USER_NODE_GITHUB_NAME = 'github_username'
+    USER_NODE_TEAM = 'team_name'
+    USER_NODE_EMPLOYEE_TYPE = 'employee_type'
+    USER_NODE_MANAGER_EMAIL = 'manager_email'
+    USER_NODE_SLACK_ID = 'slack_id'
+    USER_NODE_IS_ACTIVE = 'is_active'
+    USER_NODE_UPDATED_AT = 'updated_at'
+
+    def __init__(self,
+                 email,  # type: str
+                 first_name='',  # type: str
+                 last_name='',  # type: str
+                 name='',  # type: str
+                 github_username='',  # type: str
+                 team_name='',  # type: str
```

Comment:
I think team could be great to be a separate node to hold all the users with relations. WDYT?

Reply:
I create a todo comment and will do once we support team in Amundsen people.